.github/workflows/ci-lai-app-test-full.yml

```diff
@@ -0,0 +1,109 @@
+name: Test lightning.app full
+
+# see: https://help.github.com/en/actions/reference/events-that-trigger-workflows
+on:
+  push:
+    branches: [master, "release/*"]
+  pull_request:
+    branches: [master, "release/*"]
+    types: [opened, reopened, ready_for_review, synchronize]  # add `ready_for_review` since draft is skipped
+    paths:
+      - "setup.cfg"  # includes pytest config
+      - ".github/workflows/ci-lai-app-test-full.yml"
+      - "src/lightning_app/**"
+      - "tests/tests_app/**"
+      - "requirements/app/**"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.head_ref }}
+  cancel-in-progress: ${{ ! (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/release/')) }}
+
+env:
+  FREEZE_REQUIREMENTS: 1
+
+jobs:
+
+  lai-cpu:
+    runs-on: ${{ matrix.os }}
+    if: github.event.pull_request.draft == false
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, macOS-11, windows-2022]
+        python-version: [3.8]
```

Comment:
don't we want to test other python versions as well?

Reply:
done